### PR TITLE
Mario/add proposal proposer

### PR DESCRIPTION
**Pull request by mariopino**
**Description:**

Include proposer in proposal page.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/governance/PageProposal.vue [hunk 1]
```diff
@@ -13,6 +13,13 @@
             {{ status.badge }}
           </span>
           <h2 class="proposal-title">{{ proposal.title }}</h2>
+          <p v-if="proposal.proposer !== `unknown`" class="proposer">
```
Comment (faboweb) [inline src/components/governance/PageProposal.vue]: To not use `<p class="proposer"` twice you can use a template:

```
<template v-if="proposal.proposer !== `unknown`">
...
</template>
<template v-else>
...
</template>

File: src/components/governance/PageProposal.vue [hunk 2]
```diff
@@ -251,6 +261,7 @@ export default {
       },
       result(data) {
         /* istanbul ignore next */
+        console.log(data)
```
Comment (faboweb) [inline src/components/governance/PageProposal.vue]:
```suggestion
```

File: tests/unit/specs/components/governance/PageProposal.spec.js [hunk 3]
```diff
@@ -71,7 +71,7 @@ describe(`PageProposal`, () => {
     expect(wrapper.element).toMatchSnapshot()
   })
 
-  it(`shows an error if the proposal couldn't be found`, () => {
+   it(`shows an error if the proposal couldn't be found`, () => {
```
Comment (faboweb) [inline tests/unit/specs/components/governance/PageProposal.spec.js]: Maybe handle a test for the unknown proposer?

Comment (mariopino) [reply tests/unit/specs/components/governance/PageProposal.spec.js]: Done!

File: src/components/governance/PageProposal.vue (hunk 2 above)
Comment (mariopino) [reply src/components/governance/PageProposal.vue]: Removed!

File: src/components/governance/PageProposal.vue [hunk 4]
```diff
@@ -13,6 +13,15 @@
             {{ status.badge }}
           </span>
           <h2 class="proposal-title">{{ proposal.title }}</h2>
+          <p class="proposer">
+            <template v-if="proposal.proposer !== `unknown`">
+              Proposed by
+              <Bech32 :address="proposal.proposer" :long-form="true" />
```
Comment (jbibla) [inline src/components/governance/PageProposal.vue]: why long form?

Comment (faboweb) [reply src/components/governance/PageProposal.vue]: This should also resolve to a validator moniker if it is a validator

Comment (mariopino) [reply src/components/governance/PageProposal.vue]: My fault, I am a complete address fan :-) I am going to remove the long-form prop and figure out how to add the moniker.

Comment (mariopino) [reply src/components/governance/PageProposal.vue]: As @faboweb suggested and to following our new `please_short_pull_requests` rule :-), let's finish this pr and create a new issue for resolve the address to a moniker if the proposer is a validator.

Comment (jbibla) [reply src/components/governance/PageProposal.vue]: `please_short_pull_requests` 💯💯💯💯

**Pull request merged**